app/build.gradle

```diff
@@ -249,3 +249,5 @@ gradle.taskGraph.beforeTask { Task task ->
 }
 
 apply plugin: 'com.google.gms.google-services'
+
+apply plugin: 'kotlin-android-extensions'
```

Comment:
Does this belong down here? I'm worried about any weird errors.

Reply:
It randomly generated there somehow but I'll move it!